desktop/core/src/desktop/api_public_urls_v1.py

```diff
@@ -114,7 +115,7 @@
   re_path(r'^storage/rename/?$', api_public.storage_rename, name='storage_rename'),
   re_path(r'^storage/move/?$', api_public.storage_move, name='storage_move'),
   re_path(r'^storage/copy/?$', api_public.storage_copy, name='storage_copy'),
-  re_path(r'^storage/upload/file/?$', api_public.storage_upload_file, name='storage_upload_file'),
+  re_path(r'^storage/upload/file/?$', filebrowser_api.UploadFileAPI.as_view(), name='storage_upload_file'),
```

Comment:
why we are not defining this function in api_public file, like earlier?

Reply:
We are slowly cleaning up the api_public file so that the specific code remains in the specific working module scope for better separation of concerns. 

The URL endpoint mapping for public APIs is still present in the single file for the time being.